File: AndroidStudioProject/Calculator/app/src/main/java/com/ms/calculator/MainActivity.java
package com.ms.calculator;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @SuppressLint("MissingInflatedId")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        LinearLayout llMain;
        EditText num1,num2;
        Button btnAdd,btnSub,btnMul,btnDiv;
        TextView result;

        llMain = findViewById(R.id.llMain);
        num1 = findViewById(R.id.num1);
        num2 = findViewById(R.id.num2);
        btnAdd = findViewById(R.id.btnAdd);
        btnSub = findViewById(R.id.btnSub);
        btnMul = findViewById(R.id.btnMul);
        btnDiv= findViewById(R.id.btnDiv);
        result = findViewById(R.id.result);

        btnAdd.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                int n1 = Integer.parseInt(num1.getText().toString());
                int n2 = Integer.parseInt(num2.getText().toString());
                int sum =n1+n2;
                result.setText("Sum:" +sum);
            }
        });

        btnSub.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                int n1 = Integer.parseInt(num1.getText().toString());
                int n2 = Integer.parseInt(num2.getText().toString());

                int sub = n1-n2;
                   result.setText("Substraction: " + sub);
            }
        });

        btnMul.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                int n1 = Integer.parseInt(num1.getText().toString());
                int n2 = Integer.parseInt(num2.getText().toString());

                int mul = n1*n2;
                result.setText("Substract: " +mul);
            }
        });

        btnDiv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                int n1= Integer.parseInt(num1.getText().toString());
                int n2 = Integer.parseInt(num2.getText().toString());

                double  div= (double) n1/n2;
                result.setText("Division: " +div);

            }
        });


    }
}

File: KotlinProject/BasicSyntax/app/src/main/java/com/ms/basicsyntax/MainActivity.kt
package com.ms.basicsyntax

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        btnAdd.setOnClickListener {
            if(edtId1.text.toString() != "" && !edtId2.text.toString().equals("")){
                val num1 = edtId1.text.toString().toInt()
                val num2 = edtId2.text.toString().toInt()

                val sum = num1+num2
                Toast.makeText(this, "Sum is: $sum", Toast.LENGTH_LONG)
                txtSum.text = "Sum is: $sum"
            }

        }


    }
}